systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -40,6 +40,21 @@ private PodUtils() { }
                     pod -> pod.getMetadata().getUid()));
     }
 
+    public static String getContainerImageNameFromPod(String podName) {
+        return kubeClient().getPod(podName).getSpec().getContainers().get(0).getImage();
+    }
+
+    public static String getContainerImageNameFromPod(String podName, String containerName) {
+        return kubeClient().getPod(podName).getSpec().getContainers().stream()
+            .filter(c -> c.getName().equals(containerName))
+            .findFirst().get().getImage();
+    }
+
+    public static String getInitContainerImageName(String podName) {
+        return kubeClient().getPod(podName).getSpec().getInitContainers().stream()
```

Comment:
There is no need to use a stream to get the first item in a list, just use `.get(0)`.